metrics2/metrics2/src/test/java/io/helidon/metrics/HelidonTimerTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2020 Oracle and/or its affiliates.
+ * Copyright (c) 2018,2021 Oracle and/or its affiliates.
```

Comment:
Please use spaces between comma and the second year.

Reply:
Yeah, I know. I had to kick the pipeline to run the job again and was delayed in submitting the second, reverting commit. Sigh.